graphjet-core/src/test/java/com/twitter/graphjet/algorithms/MockEdgeTypeMask.java

```diff
@@ -0,0 +1,30 @@
+package com.twitter.graphjet.algorithms;
```

Comment:
could you move this MockEdgeTypeMask to graphjet-core/src/test/java/com/twitter/graphjet/bipartite/segment? There is another one there already HigherBitsEdgeTypeMask.

Also, please add copyRight statement here too.